monai/bundle/scripts.py

```diff
@@ -1501,3 +1527,16 @@ def init_bundle(
         copyfile(str(ckpt_file), str(models_dir / "model.pt"))
     elif network is not None:
         save_state(network, str(models_dir / "model.pt"))
+
+
+def push_to_hf_hub(bundle_dir: str, repo_name: str) -> None:
+    """
+    Push the current bundle to the Hugging Face Hub.
+
+    Args:
+        bundle_dir: path to the bundle directory to push
+        repo_name: name of the repo to create or push to the HF Hub
+    """
+    hf_api = huggingface_hub.HfApi()
+    repo_id = hf_api.create_repo(name=repo_name, exist_ok=True)
```

Comment:
```suggestion
    repo_url = hf_api.create_repo(name=repo_name, exist_ok=True)
    repo_id = repo_url.repo_id
```

The returned value of `create_repo` is a `RepoUrl` object which is a string with some special attributes like `repo_id` and `repo_type`.